### ISSUE TITLE
Should we package Discourse beta in nixpkgs?

### ISSUE THREAD

**Issue by ryantm** [NixOS/nixpkgs#146308]
Currently we package the Discourse Stable branch in Nixpkgs. Some questions I'd like to discuss:

1. Should we instead only package the Beta branch?
2. Or, should we package both? (If yes, should we maintain parallel plugins for both branches?)
3. Or, should we keep only packaging Stable?

## Some background

* An official description of the Discourse branches: https://meta.discourse.org/t/updates-always-come-before-release-notes/88919/7
* Upstream recommends following the Beta branch as a good default https://meta.discourse.org/t/please-dont-pressure-self-installers-to-be-on-beta-branch/32237/2?u=ryantm
  > You are free to choose what’s right for you, but we recommend that people follow the betas unless they are extremely risk averse.
* Beta is the default branch you get when you install Discourse from with Docker.
* All the professionally hosted (by upstream) instances of Discourse follow the Beta branch.
* https://discourse.nixos.org is using Discourse Beta branch. 
* I am also using Discourse Beta branch in my deployments.

## Discussion

I think we should focus our efforts on only packaging the Beta branch to more closely follow upstream and what we are deploying. I'd be in favor of having instructions for how someone could package the Stable branch themselves, but I wouldn't want to keep maintaining that version personally.

cc @talyz @dpausp

**Comment by talyz** [comment]
I'm fine with packaging Beta instead, since that seems to be what most other projects would consider their stable version anyway. I don't see any reason to package both - that would be a lot of work and would potentially require version checking in the module, etc.

**Comment by dpausp** [comment]
Yes, makes sense to just package beta versions. I have been using them for some years without serious problems.
